seed/models/building_file.py

```diff
@@ -305,7 +305,7 @@ def process(self, organization_id, cycle, property_view=None):
                     MeterReading(
                         start_time=mr.get('start_time'),
                         end_time=mr.get('end_time'),
-                        reading=mr.get('reading'),
+                        reading=float(mr.get('reading', 0)) * meter_conversions.get(mr.get('source_unit'), 1.00),
```

Comment:
We store meter data as kbtu in the database and convert on the way out

Reply:
ah, good to know!  makes sense too, thanks for the catch